.circleci/config.yml

```diff
@@ -29,9 +29,47 @@ jobs:
             make -C doc html
       - store_artifacts:
           path: /root/project/doc/_build/html
+      - persist_to_workspace:
+          root: doc/_build
+          paths: html
 
+  deploy_docs:
+    docker:
+      - image: node:8.10.0
+    steps:
+      - checkout
+      - attach_workspace:
+          at: doc/_build
+      - run:
+          name: Install and configure dependencies
+          command: |
+            npm install -g --silent gh-pages@2.2
+            git config user.email "CircleCI@bot.com"
+            git config user.name "CircleCI"
+      - add_ssh_keys:
+          fingerprints:
+            - "82:33:42:51:b6:32:9d:42:b3:c2:d4:a0:d5:3e:73:11"
```

Comment:
this needs to be updated once we have a real fingerprint (current one is fake)

Reply:
```suggestion
            - "b1:24:94:8c:32:93:b9:46:48:42:77:1e:00:dc:d2:d"
```